### DISCUSSION TITLE
Tools Menu - Hott Settings etc.

### DISCUSSION THREAD

**Discussion by onki69** [EdgeTX/edgetx#1177] [General]
Hi,

I am using the nightlies and missing the HoTT-functions in the Tools menu.
The only item visible is the spectrum scan option.

Is this work in progress or do I need to copy some old files?

Best regards
Onki

**Comment by pfeerick** [comment]
HoTT sounds like one of the Lua support scripts for the MultiProtocol Module. You can download the current version of them from https://downloads.multi-module.org (zip file with all the scripts is the last entry in the list).

**Comment by onki69** [comment]
Thanks - this was missing since I formatted my SD when i moved from OTX to EdgeTX.
